### PR TITLE
fix classificatino quickstart

### PR DESCRIPTION
**Pull request by avinashmaheshjoshi**
Removed usage of internal scripts

Fixed some language and typos

### CODE REVIEW

File: quickstart/monitoring/monitoring-quickstart.ipynb [hunk 1]
```diff
@@ -71,16 +71,9 @@
...
-    "AUTH_TOKEN = \"<replace with your token>\""
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "32570cab",
-   "metadata": {},
-   "source": [
-    "### Step 2: Install packages"
+    "TRUERA_URL = \"https://app.truera.net\"\n",
```
Comment (saketbairoliya2) [inline quickstart/monitoring/monitoring-quickstart.ipynb:74]: Are we using `app-demo.truera.net` or `app.truera.net`

File: quickstart/monitoring/monitoring-quickstart.ipynb [hunk 2]
```diff
@@ -101,7 +94,7 @@
    "id": "51eb099a",
    "metadata": {},
    "source": [
-    "You can download the TruEra Python SDK via [PyPI](https://pypi.org/project/truera/) with `pip install truera`.\n",
+    "You can download the TruEra Python SDK via [PyPI](https://pypi.org/project/truera/) with `pip install truera==12.6.0rc0`.\n",
```
Comment (saketbairoliya2) [inline quickstart/monitoring/monitoring-quickstart.ipynb]: This quickstart will need an update after each truera package update, that is a dependency.

**Review by saketbairoliya2 — APPROVED** [review]
Left couple of comments but otherwise LGTM.

File: quickstart/monitoring/monitoring-quickstart.ipynb (hunk 1 above)
Comment (avinashmaheshjoshi) [reply quickstart/monitoring/monitoring-quickstart.ipynb:74]: The one we share with the client will have app-demo this is pointing to app.truera.net as this is public.

File: quickstart/monitoring/monitoring-quickstart.ipynb (hunk 2 above)
Comment (avinashmaheshjoshi) [reply quickstart/monitoring/monitoring-quickstart.ipynb]: Fixed this

**Pull request merged**